addons/mail/static/src/components/message/message.xml

```diff
@@ -91,6 +107,7 @@
                                         Anonymous
                                     </strong>
                                 </t>
+                                <t t-if="messageView.isInChatWindowAndIsAlignedRight" t-call="mail.MessageViewNotification"/>
```

Comment:
It's the opposite: when message is aligned on the right, the envelope should be shown on the right of duration, so after in template (top-down in XML = left-right in UI)

Reply:
hey @alexkuhn, 

I have implemented the point you made in previous message. Did I interpret wrong? 
Correct me if i am wrong :smiley: ?

> Make sure to take alignment position of the message into account:
> 
> If message aligned on the left: date then envelope (from left to right)
> If message aligned on right: envelope then date (from left to right)

Reply:
before my fix it looked like this 

![Screenshot from 2023-09-20 10-56-26](https://github.com/odoo/odoo/assets/80447598/d3b4f944-df1d-4520-ad99-acdad40e0edd)

Reply:
> hey @alexkuhn,
> 
> I have implemented the point you made in previous message. Did I interpret wrong? Correct me if i am wrong 😃 ?
> 
> > Make sure to take alignment position of the message into account:
> > If message aligned on the left: date then envelope (from left to right)
> > If message aligned on right: envelope then date (from left to right)

Oops, I think I should read myself twice before sending the message!

Reply:
> before my fix it looked like this
> 
> ![Screenshot from 2023-09-20 10-56-26](https://user-images.githubusercontent.com/80447598/270641964-d3b4f944-df1d-4520-ad99-acdad40e0edd.png)

It looks good when the message is aligned on the right, but if you have a screenshot with message aligned on left, you'll see the envelope icon is misplaced